### PR TITLE
Convert SCSS to bootstrap classes

### PR DESCRIPTION
**Pull request by lordlabakdas**
The PR addresses the following changes:
- identify all existing SCSS elements in the code example `src/index.scss`
- convert these instances to bootstrap classes

2 Files are affected by the change
- `src.index.scss`
- `_styles.scss`

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @lordlabakdas!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.


:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://openedx.org/cla)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.     If you've signed an agreement in the past, you may need to re-sign.  See     [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/)     for details.

**Comment by natabene** [comment]
@lordlabakdas Thank you for your contribution. Once your CLA forms have been processed, we will review your PR.

**Comment by lordlabakdas** [comment]
Hi @natabene 
I intended to open a pull request with the master of my branch and mistakenly opened one against openedx master.
I can go ahead and close this one if thats ok with you.

**Comment by natabene** [comment]
@lordlabakdas Thank you for letting me know. No worries, I can cloe.

**Comment by openedx-webhooks** [comment]
@lordlabakdas Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-program-console&entry.867055334=https://github.com/openedx/frontend-app-program-console/pull/57&entry.1484655318=https://github.com/lordlabakdas&entry.752974735=2022-08-04+14:45&entry.1917517419=2022-08-08+13:20&entry.2133058324=No) so we can improve your experience in the future.

**Comment by lordlabakdas** [comment]
@natabene After a discussion about github flow internally, this is a valid PR and would kindly request your help in reopening the PR.

**Comment by natabene** [comment]
@lordlabakdas Please fill out the [forms](https://openedx.org/cla) as well, if you have not done so yet.

File: src/index.scss [hunk 1]
```diff
@@ -1,29 +0,0 @@
-$fa-font-path: "~font-awesome/fonts";
-@import "~font-awesome/scss/font-awesome";
-
-@import "~@edx/brand/paragon/fonts";
-@import "~@edx/brand/paragon/variables";
-@import "~@edx/paragon/scss/core/core";
-@import "~@edx/brand/paragon/overrides";
-
-@import "~@edx/frontend-component-header/dist/index";
-@import "~@edx/frontend-component-footer/dist/footer";
```
Comment (xitij2000) [inline src/index.scss:10]: Please don't remove these lines. We don't want to remove dependence on bootstrap and paragon, just dependance on bootstrap and paragon variables and functions.

File: src/console/_style.scss [hunk 2]
```diff
@@ -1,27 +0,0 @@
...
-  align-items: stretch;
-  flex-wrap: wrap;
-
-  .btn {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    height: 100%;
-    width: 100%;
-  }
```
Comment (xitij2000) [inline src/console/_style.scss:26]: These are applying to an element with the `btn` class inside an `enrollment-row`. You've applied these directly to the enrollment-row.

File: src/index.jsx [hunk 3]
```diff
@@ -15,7 +15,6 @@ import { AppProvider, ErrorPage } from '@edx/frontend-platform/react';
 import appMessages from './i18n';
 import configureStore from './store';
 
-import './index.scss';
```
Comment (xitij2000) [inline src/index.jsx:18]: This shouldn't be removed.

File: src/console/_style.scss [hunk 4]
```diff
@@ -1,27 +0,0 @@
...
-    width: 100%;
-    opacity: 0;
-    top: 0;
-    left: 0;
-    cursor: pointer;
-  }
-
-.collapsible-trigger {
-  background-color: theme-color("primary", "background");
-}
```
Comment (xitij2000) [inline src/console/_style.scss:13]: I don't see a replacement for this.

File: src/index.scss [hunk 5]
```diff
@@ -1,29 +0,0 @@
...
-  display: flex;
-  flex-direction: column;
-  & > main {
-    flex-grow: 1;
-  }
-}
-
-footer a img {
-  max-height: 50px;
-}
```
Comment (xitij2000) [inline src/index.scss:29]: I don't see a replacement for this.

**Review by xitij2000 — CHANGES_REQUESTED** [review]

File: src/console/_style.scss (hunk 4 above)
Comment (lordlabakdas) [reply src/console/_style.scss:13]: @xitij2000 I did not see the collapsible trigger class being used inside of code.

File: src/index.scss (hunk 5 above)
Comment (lordlabakdas) [reply src/index.scss:29]: @xitij2000 As far as I could gather, I could not see the use of this inside of code.

File: src/console/_style.scss (hunk 4 above)
Comment (xitij2000) [reply src/console/_style.scss:13]: Yes, it is not used directly by code, but it's used by components that are used by this code indirectly. So you cannot remove it without a replacement.

Comment (lordlabakdas) [reply src/console/_style.scss:13]: @xitij2000 
If I understand correctly, do you mean to say that the `collapsible-trigger` class is being used by components outside of this repo?

Inside this repo, as far as I can tell, the `collapsible-trigger` exists in test files.

**Comment by natabene** [comment]
@lordlabakdas @xitij2000 Is this PR still being pursued?

**Comment by xitij2000** [comment]
> @lordlabakdas @xitij2000 Is this PR still being pursued?

Sorry for the late response. No, it is not still being pursued.

**Comment by openedx-webhooks** [comment]
@lordlabakdas Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-program-console&entry.867055334=https://github.com/openedx/frontend-app-program-console/pull/57&entry.1484655318=https://github.com/lordlabakdas&entry.752974735=2022-08-04+14:45&entry.1917517419=2022-09-12+12:56&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**